native/ex_tokenizers/src/tokenizer.rs

```diff
@@ -27,19 +28,19 @@ impl ExTokenizersTokenizer {
     }
 }
 
-#[rustler::nif]
+#[rustler::nif(schedule = "DirtyCpu")]
```

Comment:
Would this not be DirtyIo? I'm assuming it makes a call out to the internet for a pre-trained tokenizer (similar to the Python impl), which would mean the CPU is largely idle for most of this?